### PR TITLE
Display Multiple Transit Fares if Present - `core-utils` changes only

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR is a subset of #294 that includes `core-utils` changes only.

### CODE REVIEW

File: packages/core-utils/src/itinerary.js [hunk 1]
```diff
@@ -521,6 +517,35 @@ export function calculateFares(itinerary) {
...
+        const fareComponent = itinerary.fare.fare[fareKey];
+        transitFares[fareKey] = getTransitFare(fareComponent);
+      });
+    }
+
+    return {
+      maxTNCFare,
+      minTNCFare,
+      tncCurrencyCode,
+      transitFares
```
Comment (binh-dam-ibigroup) [inline packages/core-utils/src/itinerary.js:534]: Probably need `transitCurrencyCode` here too?

File: packages/core-utils/src/itinerary.js [hunk 2]
```diff
@@ -521,6 +517,35 @@ export function calculateFares(itinerary) {
...
+    };
+  }
+
+  // Extract fare total from itinerary fares.
+  const fareComponent =
+    itinerary.fare && itinerary.fare.fare && itinerary.fare.fare.regular;
```
Comment (binh-dam-ibigroup) [inline packages/core-utils/src/itinerary.js:540]: Add FIXME for specifying what the "default" fare component in cases it is not `regular`.

File: packages/core-utils/src/itinerary.js [hunk 3]
```diff
@@ -495,18 +495,14 @@ export function getTransitFare(fareComponent) {
  * For an itinerary, calculates the transit/TNC fares and returns an object with
  * these values, currency info, as well as string formatters.
  * It is assumed that the same currency is used for transit and TNC legs.
+ *
+ * multiple being set to true will change the output behavior:
```
Comment (binh-dam-ibigroup) [inline packages/core-utils/src/itinerary.js:499]: Ok for now, but add FIXME for unifying the returned data structure.

**Review by binh-dam-ibigroup — APPROVED** [review]
Approving with the addition of FIXMEs.

File: packages/core-utils/src/itinerary.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/core-utils/src/itinerary.js:556]: Large comment added in 7f5f94945d6ceae3eaf1feeaf966f92b92dba660

File: packages/core-utils/src/itinerary.js (hunk 3 above)
Comment (miles-grant-ibigroup) [reply packages/core-utils/src/itinerary.js:499]: Have added a detailed FIXME in 7f5f94945d6ceae3eaf1feeaf966f92b92dba660

**Pull request merged**